packages/charts-components/templates/angular/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "charts-components-angular-template",
+  "version": "0.0.0",
+  "scripts": {
+    "ng": "ng",
+    "start": "yarn copy && ng serve",
+    "build": "yarn copy && ng build",
+    "watch": "yarn copy && ng build --watch --configuration development",
+    "test": "ng test",
+    "copy": "copy-folder ./node_modules/@arcgis/charts-components-angular ./charts-components-angular"
```

Comment:
this copy is awkward in the yarn workspaces / monorepo set up, because node_modules is in the root directory, but it should work for somebody downloading the folder stand-alone. something we should improve soon, but not sure there's a solution for now.

Reply:
I agree, I did my testing pulling each folder stand alone as well but for the future I will look into this.